.github/workflows/ci_macos.yml

```diff
@@ -113,13 +117,13 @@ jobs:
           CCACHE_BASEDIR: ${{ github.workspace }}
           CCACHE_DIR: ${{ github.workspace }}/.ccache
           CCACHE_COMPRESS: true
-          CCACHE_COMPRESSLEVEL: 6
-          CCACHE_MAXSIZE: 50M
+          CCACHE_COMPRESSLEVEL: 12
+          CCACHE_MAXSIZE: 75M
         run: |
-          ccache -p || true
+          ccache -z
```

Comment:
Here you wrote `ccache -z` further down (ci_misc.yml) it is `ccache -z || true`.
1. Why are they different?
2. and wouldn't be better to have `ccache -z` and `ccache -p`?

Reply:
In the linux/macos CI, we always install `ccache`.
In the misc CI, we only install it if needed (require_ccache in the job matrix)

I never found `ccache -p` useful in the CI, I found it more useful to reset statistics (`-z`) and then print the verbose statistics at the end (`-sv`).